lockbox-ios/FxA/FxAPresenter.swift

```diff
@@ -83,19 +84,27 @@ class FxAPresenter {
         }
     }
 
-    init(session: URLSession = URLSession.shared, keyManager:KeyManager = KeyManager(), keychainManager:KeychainManager = KeychainManager()) {
-        self.keyManager = keyManager
-        self.keychainManager = keychainManager
+    init(session: URLSession? = URLSession.shared,
```

Comment:
why is session optional here?

Reply:
that's so that there can be one initializer that handles both the default options for these variables (how it will be called when actually getting initialized) and dependency injection, given that there's no other nice way I've found to do mocking / dependency injection in Swift.